frontend/fingertips-frontend/components/forms/SearchForm/searchActions.test.ts

```diff
@@ -0,0 +1,37 @@
+import { expect } from '@jest/globals';
+import { searchIndicator, SearchFormState } from './searchActions';
+import { mockDeep } from 'jest-mock-extended';
+import { redirect } from 'next/navigation';
+
+jest.mock('next/navigation');
+const redirectMock = jest.mocked(redirect);
+
+function* iteratorFromList<T>(list: T[]): IterableIterator<T> {
+  for (const item of list) {
+    yield item;
+  }
+}
+
+export const getMockFormData = (formData: Record<string, string>) =>
+  mockDeep<FormData>({
+    entries: jest.fn().mockImplementation(() => {
+      const formDataEntries = Object.entries(formData);
+
+      return iteratorFromList(formDataEntries);
+    }),
+    get: (key: string) => formData[key],
+  });
+
+const initialState: SearchFormState = {
+  indicator: 'boom',
+};
```

Comment:
Given this `initialState` doesn't actually drive anything it might be worth setting the value to something different to the expected indicator ID in the redirect result. Just to rule out the function using this value instead of the one from `formData`.

Reply:
yeah that's a good spot. Will update 👍